class_character.py

```diff
@@ -0,0 +1,83 @@
+import csv
+
+
+class Character:
+    max_health = 100
+
+    def __init__(self, occupation: str, first_name: str, last_name: str, age: int):
+
+        self.features = {}
+        with open(f"./CHARACTERS/{occupation}.txt", 'r') as file:
+            reader = csv.reader(file)
+            for row in reader:
+                if row:
+                    key, feature = row
+                    self.features[key] = feature
+
+        self.first_name = first_name
+        self.last_name = last_name
+        self.full_name = ' '.join([self.first_name, self.last_name])
+        self.bio = self.features["bio"]
+
+        self.occupation = occupation
+        self.health = self.max_health
+        self.age = age
+
+        if self.features["items"]:
+            self.items = [item.replace('_', ' ') for item in self.features["items"].split(' ')]
+        else:
+            self.items = []
```

Comment:
29 linijka to taka pułapka pythonowa. Trochę mi brakuje słów żeby to wyjaśnić ale generalnie atrybuty klas w pythonie są ewaluowane raz przy ich 'wczytaniu' przez interpreter języka. Czyli ten program, który pod maską czyta i wykonuje Twój kod. O ile instrukcja if jest zależna teraz od zewnętrznych źródeł i to list comprehension będzie zawsze zwracało inną listę, to już else będzie zawsze zwracał _TĄ SAMĄ_ listę. Czyli każdy bohater z pustym ekwipunkiem będzie miał ten sam pusty ekwipunek. I gdybyś miał ich dwóch i jeden by podniósł przedmiot, to drugi też by go miał.